scout/adapter/mongo/omim.py

```diff
@@ -101,45 +110,41 @@ def omim_to_genes(self, omim_obj):
             gene_objs = [self.hgnc_gene_caption(hgnc_id) for hgnc_id in omim_obj.get("genes", [])]
         return gene_objs
 
-    def disease_term(self, disease_identifier):
-        """Return a disease term
-
-        Checks if the identifier is a disease number or a id
-
-        Args:
-            disease_identifier(str)
-
-        Returns:
-            disease_obj(dict)
-        """
+    def disease_term(
+        self,
+        disease_identifier: Union[str, int],
+        filter_project: Optional[dict] = DISEASE_FILTER_PROJECT,
```

Comment:
👍 
I agree, since it has been off before, but I say that calls for a line or two in the docstring. It is not so common (yet) that we have the methods return partial projections by default.

Reply:
Done